__tests__/app.test.js

```diff
@@ -51,4 +53,39 @@ describe('GET: /api', ()=> {
         })
     })
 })
+describe('GET /api/articles/:article_id', () => {
+    test('GET 200: returns a single article object', () => {
+        return request(app)
+        .get('/api/articles/4')
+        .expect(200)
+        .then(({body}) => {
+            const article = body
+            expect(typeof article).toBe('object')
+            expect(Array.isArray(article)).toBe(false)
+        })
+    })
+    test('GET 200: returns a single article object, corresponding to the correct id with all the correct properties and data types', () => {
+        return request(app)
+        .get('/api/articles/4')
+        .expect(200)
+        .then(({body}) => {
+            const article = body
+            expect(article.article_id).toEqual(4)
+            expect(typeof article.author).toBe('string')
+            expect(typeof article.title).toBe('string')
+            expect(typeof article.body).toBe('string')
+            expect(typeof article.topic).toBe('string')
+            expect(typeof article.created_at).toBe('string')
+            expect(typeof article.votes).toBe('number')
+            expect(typeof article.article_img_url).toBe('string')
+        })
```

Comment:
You  could use a flexible matcher eg expect.objectContaining or .toMatchObject - because this allows you to partially check objects. This will keep your tests flexible if the test data was to change or extend in properties in the future 👍